client/task-list/tasks/payments/wcpay/is-supported.js

```diff
@@ -0,0 +1,10 @@
+export function isWCPaySupported( countryCode ) {
+	const supportedCountries = [ 'US', 'PR' ];
+	if (
+		window.wcAdminFeatures &&
+		window.wcAdminFeatures[ 'wcpay/support-international-countries' ]
+	) {
+		supportedCountries.push( 'AU', 'CA', 'GB', 'IE', 'NZ' );
+	}
+	return supportedCountries.includes( countryCode );
+}
```

Comment:
Separated this out, as we use it in multiple places. I decided not to pass in the cbd industry part, although could be convinced otherwise.